cli/http_cache.rs

```diff
@@ -141,7 +141,7 @@ impl HttpCache {
     let path_part = get_url_path_and_query(url);
     let hashed_filename = crate::checksum::gen(vec![path_part.as_bytes()]);
     manifest_map.insert(path_part, hashed_filename);
-    let serialized_manifest = serde_json::to_string(&manifest_map)?;
+    let serialized_manifest = serde_json::to_string_pretty(&manifest_map)?;
```

Comment:
I really don't get why... this doesn't seem useful. It only makes parsing slower because slightly more bytes have to be handled. The *.headers.json files aren't pretty printed either afaik.

Reply:
Yeah, you're right; gonna revert it